src/OpenFeature/Model/EvaluationContextBuilder.cs

```diff
@@ -125,6 +138,23 @@ public EvaluationContextBuilder Set(string key, DateTime value)
         /// <returns>This builder</returns>
         public EvaluationContextBuilder Merge(EvaluationContext context)
         {
+            string newTargetingKey = "";
+
+            if (TargetingKey != null && TargetingKey.Trim() != string.Empty)
```

Comment:
```suggestion
            if (!string.IsNullOrWhitespace(TargetingKey))
```

Reply:
`IsNullOrWhitespace` is the best improvement .NET made on Java, change my mind.